app/controllers/wpcc/your_results_controller.rb

```diff
@@ -1,22 +1,33 @@
 module Wpcc
   class YourResultsController < EngineController
+    attr_reader :schedule, :salary_frequency
```

Comment:
Schedule and salary frequency are private methods inside the controller. Is the attr_reader are doing any difference? Because on this case, attr_reader adds two methods ('schedule' and 'salary_frequency"), which you override in the bottom of the controller. Perhaps my understand is wrong, but is this line working?